cylc/flow/network/schema.py

```diff
@@ -1398,45 +1404,43 @@ class Flow(String):
 
 # Mutations:
 
-# TODO: re-instate:
-# - get-broadcast (can just use GraphQL query BUT needs CLI access too)
-# - expire-broadcast
-
 class Broadcast(Mutation):
     class Meta:
         description = sstrip('''
-            Override or add new `[runtime]` configurations in a running
-            workflow.
+            Override `[runtime]` configurations in a running workflow.
 
             Uses for broadcast include making temporary changes to task
             behaviour, and task-to-downstream-task communication via
             environment variables.
 
-            A broadcast can target any [runtime] namespace for all cycles or
-            for a specific cycle. If a task is affected by specific-cycle and
-            all-cycle broadcasts at once, the specific takes precedence. If
-            a task is affected by broadcasts to multiple ancestor
-            namespaces, the result is determined by normal [runtime]
-            inheritance. In other words, it follows this order:
-
-            `all:root -> all:FAM -> all:task -> tag:root -> tag:FAM ->
-            tag:task`
-
-            Broadcasts persist, even across restarts, until they expire
-            when their target cycle point is older than the oldest current in
-            the workflow, or until they are explicitly cancelled with this
-            command.  All-cycle broadcasts do not expire.
-
-            For each task the final effect of all broadcasts to all namespaces
-            is computed on the fly just prior to job submission.  The
-            `--cancel` and `--clear` options simply cancel (remove) active
-            broadcasts, they do not act directly on the final task-level
-            result. Consequently, for example, you cannot broadcast to "all
-            cycles except Tn" with an all-cycle broadcast followed by a cancel
-            to Tn (there is no direct broadcast to Tn to cancel); and you
-            cannot broadcast to "all members of FAMILY except member_n" with a
-            general broadcast to FAMILY followed by a cancel to member_n (there
-            is no direct broadcast to member_n to cancel).
+            A broadcast can set/override any `[runtime]` configuration for all
+            cycles or for a specific cycle. If a task is affected by a
+            specific-cycle and an all-cycle broadcasts at the same time, the
+            specific takes precedence.
+
+            Broadcasts can also target all tasks, specific tasks or families of
+            tasks. If a task is affected by broadcasts to multiple ancestor
+            namespaces (tasks it inherits from), the result is determined by
+            normal `[runtime]` inheritance. In other words, it follows this
+            order:
+
+            ```
+            all:root -> all:FAM -> all:task -> tag:root -> tag:FAM -> tag:task
+            ```
```

Comment:
Unfortunately this overflows the mutation form: 

<img src="https://user-images.githubusercontent.com/61982285/176715926-b9abc405-8dd2-4674-89fb-e24c6206735d.png" width=500>

```suggestion
            `all:root -> all:FAM -> all:task -> tag:root -> tag:FAM ->
            tag:task`
```

Reply:
Decided to remove this block.